composer.json

```diff
@@ -43,19 +43,20 @@
     "require": {
         "php": "^7.3",
         "ext-phar": "*",
+        "composer-plugin-api": "^1.0 || ^2.0",
         "amphp/parallel-functions": "^0.1.3",
         "beberlei/assert": "^3.2",
         "composer/package-versions-deprecated": "^1.8",
         "composer/semver": "^3.2",
         "composer/xdebug-handler": "^1.3.2",
-        "composer-plugin-api": "^1.0 || ^2.0",
         "hoa/compiler": "^3.17",
```

Comment:
`hoa/compiler` could be removed now, right?

Reply:
removed